adopterAPI.json

```diff
@@ -354,40 +427,152 @@
-          0,
-          1,
-          2
+          "Active",
+          "Blocked",
+          "Deleted"
         ],
-        "type": "integer",
-        "format": "int32",
-        "description": "0 - Active, 1 - Blocked, 2 - Deleted"
+        "type": "string"
       },
-      "NewAddress": {
+      "AdopterStatusDTO": {
+        "enum": [
+          "Active",
+          "Blocked",
+          "Deleted"
+        ],
+        "type": "string"
+      },
+      "AnnouncementDTO": {
         "type": "object",
         "properties": {
-          "street": {
-            "type": "string"
+          "id": {
+            "type": "string",
+            "format": "uuid"
           },
-          "city": {
-            "type": "string"
+          "title": {
+            "type": "string",
+            "nullable": true
           },
-          "province": {
-            "type": "string"
+          "description": {
+            "type": "string",
+            "nullable": true
           },
-          "postalCode": {
-            "type": "string"
+          "creationDate": {
+            "type": "string",
+            "format": "date-time"
           },
-          "country": {
-            "type": "string"
+          "closingDate": {
+            "type": "string",
+            "format": "date-time",
+            "nullable": true
+          },
+          "lastUpdateDate": {
+            "type": "string",
+            "format": "date-time"
+          },
+          "status": {
+            "$ref": "#/components/schemas/AnnouncementStatusDTO"
+          },
+          "pet": {
+            "$ref": "#/components/schemas/PetDTO"
+          }
+        },
+        "additionalProperties": false
+      },
+      "AnnouncementStatusDTO": {
+        "enum": [
+          "Open",
+          "Closed",
+          "DuringVerification",
+          "Deleted"
+        ],
+        "type": "string"
+      },
+      "ApplicationDTO": {
+        "type": "object",
+        "properties": {
+          "id": {
+            "type": "string",
+            "format": "uuid"
+          },
+          "creationDate": {
+            "type": "string",
+            "format": "date-time",
+            "nullable": true
+          },
+          "lastUpdateDate": {
+            "type": "string",
+            "format": "date-time",
+            "nullable": true
+          },
+          "announcementId": {
+            "type": "string",
+            "format": "uuid"
+          },
+          "announcement": {
+            "$ref": "#/components/schemas/AnnouncementDTO"
+          },
+          "adopter": {
+            "$ref": "#/components/schemas/AdopterDTO"
+          },
+          "applicationStatus": {
+            "$ref": "#/components/schemas/ApplicationStatusDTO"
+          }
+        },
+        "additionalProperties": false
+      },
+      "ApplicationStatusDTO": {
+        "enum": [
+          "Created",
+          "Accepted",
+          "Rejected",
+          "Withdrawn",
+          "Deleted"
```

Comment:
Do we need `"Deleted"` here? When would we use this? I don't think we can set it from API.

Reply:
I think ``"Deleted"`` is identical to ``"Withdrawn"`` according to the documentation and can be safely deleted. Someone correct me if I'm wrong, please.

Reply:
It was supposed to be used by administrator to soft-delete application when it was reported